.github/workflows/docker-publish-lakefs-rclone-export.yaml

```diff
@@ -37,6 +37,9 @@ jobs:
           aws-region: us-east-1
           role-to-assume: ${{ secrets.AWS_ROLE_TO_ASSUME }}
 
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
```

Comment:
Will this cause [Build and push image](https://github.com/treeverse/lakeFS/pull/5770/files#diff-58b23127f3a388794f61ddd4c8241689d407c643ae970e8b7999124461726f44L50-L51) to buildx??

Reply:
It should use `docker buildx` instead of `docker build` for the image build step.